src/ontology/bridge/cl-bridge-to-ehdaa2.owl

```diff
@@ -131,12 +137,18 @@
                 <owl:intersectionOf rdf:parseType="Collection">
                     <rdf:Description rdf:about="http://purl.obolibrary.org/obo/CL_0002239"/>
                     <owl:Restriction>
-                        <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/BFO_0000050"/>
+                        <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/RO_0002162"/>
```

Comment:
Expected consequence of #3466, everything’s fine.

Likewise for all other bridge files.